debvader/debvader.py

```diff
@@ -8,12 +8,15 @@
 import scipy
 from scipy import optimize
 
-import skimage   
 from skimage import metrics
 import sep
 
-from debvader import model
-
+#from debvader import model
+###### TO SUPRESSS AND UNCOMMENT PREVIOUS LINES
+import sys
+sys.path.insert(0,'.')
+import model
+######
```

Comment:
Whaaaat ?!

Reply:
Old version. Modified in commit [b085507](https://github.com/BastienArcelin/debvader/commit/b085507933267ac5fdcff4a486cde70d6985bc94#diff-a5dae2bb532104d1ce10a8b52d5c651b023b7d0d57e87a12c0c6942e0d44cee7)